data-raw/layers_df.csv

```diff
@@ -0,0 +1,23 @@
+layer_name,title,record,resource, local,using_shortcuts
+bc_cities,BC Major Cities Points, b678c432-c5c1-4341-88db-0d6befa0c7f8,443dd858-2e37-4a8f-937a-f53359f16e64,FALSE,TRUE
+bc_bound_hres,High Resolution BC Boundary,30aeb5c1-4285-46c8-b60b-15b1a6f4258b,3d72cf36-ab53-4a2a-9988-a883d7488384,TRUE,FALSE
```

Comment:
`bc_bound_hres`, `bc_bound`, and `bc_neighbours` should now be `FALSE` for `internal` right? But still be `FALSE` for `using_shortcuts`?

Reply:
Do you mean `local`? Whatever the case do we actually still need that?

Reply:
Oh yes I mean `local`, sorry. I honestly didn't notice if that field was being used, just that your recent commits moved those layers from local to remote...